tscat_gui/utils/keyword_list.py

```diff
@@ -0,0 +1,172 @@
+from .flow_layout import FlowLayout
+
+from PySide2 import QtWidgets, QtCore, QtGui
+
+# UTF-8 letters in the begging, then also numbers and underscore
+_tag_validation_regex = QtCore.QRegularExpression(r'\p{L}\w*')
+
+
+class _Keyword(QtWidgets.QFrame):
+    updated = QtCore.Signal(QtWidgets.QWidget)
+    closed = QtCore.Signal(QtWidgets.QWidget)
+
+    class Label(QtWidgets.QLabel):
+        clicked = QtCore.Signal()
+
+        def __init__(self, text: str, parent=None):
+            super().__init__(text, parent)
+
+        def mousePressEvent(self, event: QtGui.QMouseEvent) -> None:
+            if event.button() == QtCore.Qt.MouseButton.LeftButton:
+                self.clicked.emit()
+            super().mousePressEvent(event)
+
+    class DeleteLabel(QtWidgets.QLabel):
+        clicked = QtCore.Signal()
+
+        def __init__(self, parent=None):
+            super().__init__('✕', parent)
+            self.setMargin(1)
+            self.setObjectName('DeleteLabel')
+            self.setStyleSheet(
+                "#DeleteLabel { border-radius: 5px; background: #444; min-width: 10px; min-height: 10px; }")
```

Comment:
not sure about this:   background: `#444`
![tags_colors](https://user-images.githubusercontent.com/7570249/153050307-85a2ad4c-7935-4906-aea4-e08d4a7a6214.png)

Reply:
`#444` is the equivalent of `#444444` . Or do you mean the color itself, We might better use color constants from QT.

Reply:
I mean, I'm not sure about a fixed color, specially here with this theme, we barely see the [x].